app/controllers/Whereuat.scala

```diff
@@ -70,6 +70,14 @@ class Whereuat extends Controller {
 
 
   // Route actions
+  def index = Action {
+    Ok(views.html.index())
+  }
+
+  def team = Action {
```

Comment:
I like the idea of having a single page app for this, but good for now. future release